build.sbt

```diff
@@ -25,7 +25,7 @@ val elephantbirdVersion = "4.15"
 val hadoopLzoVersion = "0.4.19"
```

Comment:
would be nice to turn mima back on in this PR or I guess the next and make sure we are compatibile with `0.17.0`.